RockSniffer/RPC/DiscordRPCHandler.cs

```diff
@@ -70,6 +71,13 @@ internal void UpdatePresence()
             //If we have a valid song and are playing a song
             if ((songdetails != null && readout != null) && (state == SnifferState.SONG_STARTING || state == SnifferState.SONG_PLAYING || state == SnifferState.SONG_ENDING))
             {
+                // Get the appropriate album cover. In case of failure display returned text.
+                {
```

Comment:
Add a configuration option in the rpc config, so users can opt out.
On the assumption that Apple collects telemetry from these calls.

Reply:
Done